### DISCUSSION TITLE
Ability to use a Zen profile or container as a profile within Safari.

### DISCUSSION THREAD

**Discussion by utopiaeh** [zen-browser/desktop#2983] [Ideas]
I’m looking for a way to use profiles in a way similar to Safari’s implementation but with features like container usage. Ideally, I’d like to switch profiles without opening a new window, and for profiles to remain isolated, much like containers do. Each profile should keep the same browser settings but allow me to control which specific settings are shared between profiles, like passwords, browser settings, plugins, and shortcuts. However, history, cookies, and other data should stay separate for each profile.

**Comment by cheeeebs** [comment]
The Zen Browser developers don't have control over what Apple or their Safari team do.

**Reply by utopiaeh** [reply]
No, the main idea is the ability to use profiles in a way similar to how it’s implemented in Safari. For example, I find it very convenient that profiles in Safari keep the same browser settings for each profile, and switching between profiles doesn’t open new windows. They are implemented like containers but remain isolated.

**Reply by cheeeebs** [reply]
Please make that clearer in the original post, people may confuse it that you want to use Zen profiles or containers in Safari. 

As for the implementation of this though, yes I would also like to have this but I feel like it would take a tremendous effort to change a core Firefox feature.
